source/admin_manual/ssl/certificate_creation.rst

```diff
@@ -31,12 +31,12 @@ The clients have a built-in list of trusted CAs, so getting it from one of those
 
 So the two options end up being three:
 
- # Get it from an external CA (which already have the public keys installed on the clients, so the clients trust the CA)
- # # Purchase the certificate.
- # # Get a free certificate.
- # Create your own CA (in this case you'll have to add the CA public keys into the clients, as you are bot trusted by default)
+#. Get it from an external CA (which already have the public keys installed on the clients, so the clients trust the CA)
```

Comment:
This and only this line shows up as bold and italicized on github, I think something is broken with github's rendering - not sure where to bring that up.

Reply:
Pretty minor.

But I looked at the html, it has "\<dt\>" around that line it so looks like a backend / github issue.

dt: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt

Here's what I see:

![image](https://github.com/dovecot/documentation/assets/42042889/f2a9428c-5369-410c-b7bb-e7321a5f2777)

html has:

```
<dt>Get it from an external CA (which already have the public keys installed on the clients, so the clients trust the CA)</dt>
```